addons/core/translations/resources/en-us.yaml

```diff
@@ -373,6 +373,9 @@ credential-store:
       help: A PEM-encoded private key matching the client certificate from Client Certificate.
     client_certificate_key_hmac:
       label: Client Certificate Key HMAC
+    worker_filter:
```

Comment:
It looks like we have an identical translation in the `forms` [translations](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/form/en-us.yaml#L56), should we just use that instead?

Reply:
Done!